src/guides/guide-beatles/beatles_guide.json

```diff
@@ -0,0 +1,69 @@
+{
+    "options": {
+        "repositoryIdBase": "the_beatles"
+    },
+    "steps": [
+        {
+            "guideBlockName": "create-repository",
+            "options": {
+                "rulesetName": "RDFS-Plus (Optimized)"
+            }
+        },
+        {
+            "guideBlockName": "select-repository-dropdown"
+        },
+        {
+            "guideBlockName": "enable-autocomplete"
+        },
+        {
+            "guideBlockName": "download-guide-resource",
+            "options": {
+                "resourcePath": "guides/guide-beatles",
+                "resourceFile": "beatles.ttl"
+            }
+        },
+        {
+            "guideBlockName": "import-rdf-file",
+            "options": {
+                "fileName": "beatles.ttl"
+            }
+        },
+        {
+            "guideBlockName": "visual-graph",
+            "options": {
+                "easyGraphInputText": "be",
+                "clickOnIRI": "http://stardog.com/tutorial/The_Beatles"
+            }
+        },
+        {
+            "guideBlockName": "visual-graph-expand",
+            "options": {
+                "clickOnIRI": "http://stardog.com/tutorial/John_Lennon",
+                "shortIRI": "John_Lennon"
+            }
+        },
+        {
+            "guideBlockName": "visual-graph-properties",
+            "options": {
+                "clickOnIRI": "http://stardog.com/tutorial/Imagine",
+                "shortIRI": "Imagine"
+            }
+        },
+        {
+            "guideBlockName": "generic-message",
+            "options": {
+                "title": {
+                    "en": "Visual graph: more",
```

Comment:
If we make translations in this way, we have to update all guides, when we add a new language. 
In my opinion better is if we use label key and a file per language, because when we add a new language we have to add only one file.